master/buildbot/config.py

```diff
@@ -127,7 +129,7 @@ def __init__(self):
         "properties", "revlink", "schedulers", "slavePortnum", "slaves",
         "status", "title", "titleURL", "user_managers", "validation", "realTimeServer",
         "analytics_code", "gzip", "autobahn_push", "lastBuildCacheDays",
-        "requireLogin", "globalFactory", "slave_debug_url",
+        "requireLogin", "globalFactory", "slave_debug_url", "slaveManagerUrl",
```

Comment:
We started talking about the inconsistency in buildbot/katana code and we should probably come back to it, for now the decision is using same standard as twisted library (camelCase), the code looks less inconsistent if we use it.

About slave vs. a new name for now I think should make a new card for refactoring slave name, since slave is used everywhere